test/cgrates_web_jsonapi_web/controllers/cdr_controller_test.exs

```diff
@@ -286,6 +286,55 @@ defmodule CgratesWebJsonapiWeb.CdrControllerTest do
 
       assert length(response) == 1
     end
+
+    test "filtering by extra fields", %{conn: conn} do
+      insert(:cdr,
+        usage: 10_000,
+        cost: 10,
+        created_at: "2015-01-23T23:50:07Z",
+        extra_fields: %{
+          "direction" => "in",
+          "cid" => "123"
+        }
+      )
+
+      cdr1 =
+        insert(:cdr,
+          usage: 10_000,
+          cost: 10,
+          created_at: "2015-02-23T22:50:07Z",
+          extra_fields: %{
+            "direction" => "out",
+            "cid" => "321"
+          }
+        )
+
+      cdr2 =
+        insert(:cdr,
+          usage: 10_002,
+          cost: 12,
+          created_at: "2015-02-23T22:50:07Z",
+          extra_fields: %{
+            "direction" => "out",
+            "cid" => "231"
+          }
+        )
+
+      conn =
+        get(conn, Routes.cdr_path(conn, :index),
+          filter: %{
+            extra_fields: %{
+              "cid" => %{op: "LIKE", val: "123"},
```

Comment:
If custom operations are not supported and you have a hardcoded "like" statement you should not send "op" key.
+ "op" => op is a unused argument for the filter function